diff --git a/V2/Model.py b/V2/Model.py
index 2baa393..d592b57 100644
--- a/V2/Model.py
+++ b/V2/Model.py
@@ -668,13 +668,13 @@ class Model:
 
 
 class Param_Tunning:
-    def __init__(self, params: dict) -> None:
+    def __init__(self,) -> None:
         f"""
         initialize the Class
         params - dict like {Model().test}
         """
 
-    def tune(self, params) -> dict:
+    def tune(self, params : dict) -> dict:
         """
         Tune all of the parameters
         """
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index 96a8a3c..fb109a5 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211031_115221-2ka0cy99
\ No newline at end of file
+run-20211031_120713-3kng0982
\ No newline at end of file
