diff --git a/V2/Model.py b/V2/Model.py
index e6523a3..a122d6c 100644
--- a/V2/Model.py
+++ b/V2/Model.py
@@ -42,10 +42,10 @@ class Model:
         base_lr: float = 0.00025,
         data: pd.DataFrame = pd.read_csv("./Data.csv").sample(frac=1),
         labels: list = ["Card"],
-        max_iter: int = 1000,
-        eval_period: int = 5,
-        ims_per_batch: int = 5,
-        batch_size_per_image: int = 256,
+        max_iter: int = 500,
+        eval_period: int = 50,
+        ims_per_batch: int = 2,
+        batch_size_per_image: int = 128,
         score_thresh_test: float = 0.625,
         model: str = "COCO-Detection/faster_rcnn_X_101_32x8d_FPN_3x.yaml",
         name: str = "baseline",
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index c586bc5..b8d7148 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211029_142202-kbbh9tez
\ No newline at end of file
+run-20211029_155621-3rkua0j4
\ No newline at end of file
