diff --git a/V2/Final.py b/V2/Final.py
deleted file mode 100644
index 2510d43..0000000
--- a/V2/Final.py
+++ /dev/null
@@ -1,4 +0,0 @@
-from Model import *
-
-model = Model()
-model.train()
diff --git a/V2/Model.py b/V2/Model.py
index 676b955..446f98b 100644
--- a/V2/Model.py
+++ b/V2/Model.py
@@ -42,14 +42,15 @@ class Model:
         base_lr: float = 0.00025,
         data: pd.DataFrame = pd.read_csv("./Data.csv").sample(frac=1),
         labels: list = ["Card"],
-        max_iter: int = 500,
+        max_iter: int = 1000,
         eval_period: int = 5,
-        ims_per_batch: int = 2,
-        batch_size_per_image: int = 128,
+        ims_per_batch: int = 5,
+        batch_size_per_image: int = 512,
         score_thresh_test: float = 0.625,
         model: str = "COCO-Detection/faster_rcnn_X_101_32x8d_FPN_3x.yaml",
         name: str = "baseline",
         create_target_and_preds: int = 29,
+        test_sample_size=32
     ) -> None:
         """
         - __init__ = initialize and get all of the params need
@@ -113,6 +114,7 @@ class Model:
         self.NAME = name
         self.cfg = self.create_cfg()  # Creating the model config
         self.create_target_and_preds_iter = create_target_and_preds
+        self.test_sample_size = test_sample_size
         self.remove_files_in_output()
 
     @staticmethod
@@ -172,7 +174,7 @@ class Model:
                 self.data = np.load(
                     "./data.npy", allow_pickle=True
                 )  # Loading already saved detectron2 format file
-                self.data = self.data[:62]
+                self.data = self.data[:self.test_sample_size] # TODO
                 return self.data
         if "data.npy" in os.listdir("./"):
             self.data = np.load("./data.npy", allow_pickle=True)
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index daad5d3..c586bc5 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211029_110514-z3t1ecay
\ No newline at end of file
+run-20211029_142202-kbbh9tez
\ No newline at end of file
