diff --git a/V2/run.py b/V2/run.py
index 67a8e01..de8b91f 100644
--- a/V2/run.py
+++ b/V2/run.py
@@ -1,4 +1,5 @@
 from Model import *
-
+model = Model()
+model.train()
 pt = Param_Tunning()
 pt.tune(params)
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index 96a8a3c..ead9f96 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211031_115221-2ka0cy99
\ No newline at end of file
+run-20211031_131557-2r4740f9
\ No newline at end of file
