diff --git a/V2/Model.py b/V2/Model.py
index 676b955..d8281af 100644
--- a/V2/Model.py
+++ b/V2/Model.py
@@ -42,7 +42,7 @@ class Model:
         base_lr: float = 0.00025,
         data: pd.DataFrame = pd.read_csv("./Data.csv").sample(frac=1),
         labels: list = ["Card"],
-        max_iter: int = 500,
+        max_iter: int = 5000,
         eval_period: int = 5,
         ims_per_batch: int = 2,
         batch_size_per_image: int = 128,
@@ -172,7 +172,7 @@ class Model:
                 self.data = np.load(
                     "./data.npy", allow_pickle=True
                 )  # Loading already saved detectron2 format file
-                self.data = self.data[:62]
+                # self.data = self.data[:62] # TODO
                 return self.data
         if "data.npy" in os.listdir("./"):
             self.data = np.load("./data.npy", allow_pickle=True)
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index daad5d3..fb8173e 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211029_110514-z3t1ecay
\ No newline at end of file
+run-20211029_142014-93jg100m
\ No newline at end of file
