diff --git a/V2/output.png b/V2/output.png
index 24f7c5a..c273773 100644
Binary files a/V2/output.png and b/V2/output.png differ
diff --git a/V2/output/last_checkpoint b/V2/output/last_checkpoint
deleted file mode 100644
index da0e9f9..0000000
--- a/V2/output/last_checkpoint
+++ /dev/null
@@ -1 +0,0 @@
-model_final.pth
\ No newline at end of file
diff --git a/V2/output/test_coco_format.json.lock b/V2/output/test_coco_format.json.lock
deleted file mode 100644
index e69de29..0000000
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index 611a059..a1abede 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211028_204457-3r95n0qd
\ No newline at end of file
+run-20211028_211134-1nu227mf
\ No newline at end of file
