diff --git a/V2/API.py b/V2/API.py
new file mode 100644
index 0000000..5d265ef
--- /dev/null
+++ b/V2/API.py
@@ -0,0 +1,7 @@
+from flask import *
+from flask_restful import API
+
+app = Flask(__name__)
+app.debug = True
+app.secret_key = 'secret_key'
+api = API(app)
diff --git a/V2/Model.py b/V2/Model.py
index 1bb9b8c..4f0e514 100644
--- a/V2/Model.py
+++ b/V2/Model.py
@@ -538,18 +538,16 @@ class Model:
         wandb.log({"MSE": mse})
         wandb.log({"PSNR": psnr})
         self.save(
-            {
-                "trainer": trainer,
-                "predictor": predictor,
-                "metrics_coco": metrics_coco,
-                "metrics_file": metrics_file,
-                "test_images": test_images,
-                "preds": preds,
-                "target": target,
-                "rmse": rmse,
-                "mse": mse,
-                "psnr": psnr,
-            }
+            trainer=trainer,
+            predictor=predictor,
+            metrics_coco=metrics_coco,
+            metrics_file=metrics_file,
+            test_images=test_images,
+            preds=preds,
+            target=target,
+            rmse=rmse,
+            mse=mse,
+            psnr=psnr,
         )
         wandb.finish()
         return {
diff --git a/V2/output/metrics.json b/V2/output/metrics.json
deleted file mode 100644
index 5b4c19d..0000000
--- a/V2/output/metrics.json
+++ /dev/null
@@ -1,2 +0,0 @@
-{"data_time": 0.0022578255002372316, "eta_seconds": 263.42584767504377, "fast_rcnn/cls_accuracy": 0.83203125, "fast_rcnn/false_negative": 1.0, "fast_rcnn/fg_cls_accuracy": 0.0, "iteration": 19, "loss_box_reg": 0.4040021449327469, "loss_cls": 0.5641954243183136, "loss_rpn_cls": 0.015174644999206066, "loss_rpn_loc": 0.010738587938249111, "lr": 1.9231e-05, "roi_head/num_bg_samples": 115.5, "roi_head/num_fg_samples": 12.5, "rpn/num_neg_anchors": 250.5, "rpn/num_pos_anchors": 5.5, "time": 1.1453297725001903, "total_loss": 1.0348275592550635}
-{"data_time": 0.003017994999936491, "eta_seconds": 227.78666137488017, "fast_rcnn/cls_accuracy": 0.900390625, "fast_rcnn/false_negative": 1.0, "fast_rcnn/fg_cls_accuracy": 0.0, "iteration": 39, "loss_box_reg": 0.36825934052467346, "loss_cls": 0.4347027987241745, "loss_rpn_cls": 0.023806994780898094, "loss_rpn_loc": 0.005458259489387274, "lr": 3.9211e-05, "roi_head/num_bg_samples": 115.5, "roi_head/num_fg_samples": 12.5, "rpn/num_neg_anchors": 252.5, "rpn/num_pos_anchors": 3.5, "time": 1.0239262004997727, "total_loss": 0.8840265450999141}
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index 78c2eb4..753a149 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211029_160844-367hqkf5
\ No newline at end of file
+run-20211029_161824-1q2twbdb
\ No newline at end of file
