diff --git a/V2/Model.py b/V2/Model.py
index 40f2e49..bf61dd8 100644
--- a/V2/Model.py
+++ b/V2/Model.py
@@ -42,8 +42,8 @@ class Model:
         base_lr: float = 0.00025,
         data: pd.DataFrame = pd.read_csv("./Data.csv").sample(frac=1),
         labels: list = ["Card"],
-        max_iter: int = 250,
-        eval_period: int = 250,
+        max_iter: int = 500,
+        eval_period: int = 500,
         ims_per_batch: int = 2,
         batch_size_per_image: int = 128,
         score_thresh_test: float = 0.625,
diff --git a/V2/output/events.out.tfevents.1635516260.programmer-rd-ai.119029.0 b/V2/output/events.out.tfevents.1635516260.programmer-rd-ai.119029.0
deleted file mode 100644
index 1995a0b..0000000
Binary files a/V2/output/events.out.tfevents.1635516260.programmer-rd-ai.119029.0 and /dev/null differ
diff --git a/V2/output/metrics.json b/V2/output/metrics.json
deleted file mode 100644
index e69de29..0000000
diff --git a/V2/run.py b/V2/run.py
index da23030..f10e013 100644
--- a/V2/run.py
+++ b/V2/run.py
@@ -1,20 +1,20 @@
 from Model import *
 models = [
         # "fast_rcnn_R_50_FPN_1x.yaml",
-        # "faster_rcnn_R_50_C4_1x.yaml",
+        "faster_rcnn_R_50_C4_1x.yaml",
         "faster_rcnn_R_50_C4_3x.yaml",
-        # "faster_rcnn_R_50_DC5_1x.yaml",
-        # "faster_rcnn_R_50_DC5_3x.yaml",
-        # "retinanet_R_50_FPN_1x.py",
-        # "retinanet_R_50_FPN_1x.yaml",
-        # "retinanet_R_50_FPN_3x.yaml",
-        # "rpn_R_50_C4_1x.yaml",
-        # "rpn_R_50_FPN_1x.yaml",
-        # "faster_rcnn_R_50_FPN_1x.yaml",
-        # "faster_rcnn_R_50_FPN_3x.yaml",
-        # "faster_rcnn_R_101_DC5_3x.yaml",
-        # "faster_rcnn_R_101_FPN_3x.yaml",
-        # "faster_rcnn_X_101_32x8d_FPN_3x.yaml",
+        "faster_rcnn_R_50_DC5_1x.yaml",
+        "faster_rcnn_R_50_DC5_3x.yaml",
+        "retinanet_R_50_FPN_1x.py",
+        "retinanet_R_50_FPN_1x.yaml",
+        "retinanet_R_50_FPN_3x.yaml",
+        "rpn_R_50_C4_1x.yaml",
+        "rpn_R_50_FPN_1x.yaml",
+        "faster_rcnn_R_50_FPN_1x.yaml",
+        "faster_rcnn_R_50_FPN_3x.yaml",
+        "faster_rcnn_R_101_DC5_3x.yaml",
+        "faster_rcnn_R_101_FPN_3x.yaml",
+        "faster_rcnn_X_101_32x8d_FPN_3x.yaml",
     ]
 max_iters = 125
 labels = ['Card']
diff --git a/V2/wandb/latest-run b/V2/wandb/latest-run
index 492bf0b..11aa79c 120000
--- a/V2/wandb/latest-run
+++ b/V2/wandb/latest-run
@@ -1 +1 @@
-run-20211029_193409-1yk5oqbk
\ No newline at end of file
+run-20211029_193525-1yjlj55k
\ No newline at end of file
diff --git a/V2/wandb/run-20211029_193409-1yk5oqbk/run-1yk5oqbk.wandb b/V2/wandb/run-20211029_193409-1yk5oqbk/run-1yk5oqbk.wandb
index 68cebd7..7b57157 100644
Binary files a/V2/wandb/run-20211029_193409-1yk5oqbk/run-1yk5oqbk.wandb and b/V2/wandb/run-20211029_193409-1yk5oqbk/run-1yk5oqbk.wandb differ
